app/views/components/details.scala.html

```diff
@@ -18,7 +18,7 @@
 
 @(summaryText: String = "")(bodyHtml: Html = HtmlFormat.empty)
 
-<details class="govuk-details details-font" data-module="govuk-details">
+<details id="tariffReference" class="govuk-details details-font" data-module="govuk-details">
```

Comment:
why `id` - we should use class in component level  ?

Reply:
This component only does one thing and it's only ever used once on a page.  It's easier to access by a unique id in the acceptance tests,

Reply:
Hmm... - OK
On the other hand - from perspective of component robustness - class would have more pros.
About API in acceptance test - maybe would be worth to access element/s in easier way then this.